src/zdns/answers_generate.go

```diff
@@ -0,0 +1,102 @@
+//go:build ignore
```

Comment:
Let's add the license and attribution linking to the miekg project.

Reply:
added in https://github.com/zmap/zdns/pull/465/commits/6f09067a5b2b0e87abf4163ec2c12709f85bb37f